misc/services/acrn_manager/README.rst

```diff
@@ -87,7 +87,7 @@ Use the ``list`` command to display VMs and their state:
 Start VM
 ========
 
-If a VM is in a ``stopped`` state, you can start it with the ``start``
+If a VM is in a stopped state, you can start it with the start command. The acrnd service automatically loads launch scripts under /usr/share/acrn/conf/add/ to boot the VMs.
```

Comment:
It would have been better to use the double-backticks for some of the text in there, e.g. `start`, `acrnd` and `/usr/share/acrn/conf/add/`.

How does `acrnd` know which launch script to use from within that directory?

Reply:
@zhangrouyu Please answer the above questions "How does acrnd know which launch script to use from within that directory?"